docs/tutorial/starting.rst

```diff
@@ -0,0 +1,22 @@
+========================
```

Comment:
Maybe this one can be moved outside of the tutorial in some kind of quick'n'dirty getting started or quickstart for the impatient.